core/src/edu/asu/c3simulator/util/PlayerInterfaceImplementation.java

```diff
@@ -0,0 +1,89 @@
+package edu.asu.c3simulator.util;
+
+import edu.asu.c3simulator.simulation.C3Simulation;
+import edu.asu.c3simulator.simulation.Player;
+import edu.asu.c3simulator.simulation.SocioeconomicStatus;
+
+/**
+ * Implementation of the Player interface Provides detail about various player
+ * attributes that are needed for PlayerStatusDisplay
+ * 
+ * @author Reigel, Justin
+ *
+ */
+
+public class PlayerInterfaceImplementation implements Player
```

Comment:
I'll mention this again: naming schemes in these instances (where an interface with the "perfect" name is being used, and we need an implementation class for it) are difficult to devise, and are a problem we should tackle as a group, or at least open a gitHub discussion for it. PlayerInterfaceImplementation, PlayerImp, etc. are generally smurfy, and I think we can do better if we collectively put our heads together. 

Once again, ArrayList and LinkedList both implement List, yet they have meaningful names that depict their interface **and** their unique aspects. HashMap, HashTable, and LinkedHashMap are other good Java examples of good naming conventions. I don't have a good proposal for this case (Player), but I'd like to open a group conversation to figure it out. 

For now, do the following so we can get this into master_testing:
* **Open an issue on GitHub to discuss the naming topic**
* Add a TODO tag on line 14. 
* Add an @version tag in the javadoc (I'd go with V0.1)
* Update the javadoc to reflect the fact that this class will be deprecated and replaced with a refactored version in the near future.